### PR TITLE
Added some class summaries

### PR DESCRIPTION
**Pull request by k-boyle**

### CODE REVIEW

File: src/Discord.Net.Commands/ModuleBase.cs [hunk 1]
```diff
@@ -4,8 +4,15 @@
...
+    /// <summary>
+    ///     The base class that command modules inherit from where the context is of type <see cref="ICommandContext"/>.
+    /// </summary>
     public abstract class ModuleBase : ModuleBase<ICommandContext> { }
 
+    /// <summary>
+    ///     The base class that command modules inherit from.
+    /// </summary>
+    /// <typeparam name="T">A class that implements <see cref="ICommandContext"/>.</typeparam>
     public abstract class ModuleBase<T> : IModuleBase
```
Comment (k-boyle) [inline src/Discord.Net.Commands/ModuleBase.cs]: Can't say I'm a fan of these summaries, thoughts?

File: src/Discord.Net.Commands/ModuleBase.cs [hunk 2]
```diff
@@ -4,8 +4,15 @@
 
 namespace Discord.Commands
 {
+    /// <summary>
+    ///     The base class that command modules inherit from where the context is of type <see cref="ICommandContext"/>.
+    /// </summary>
     public abstract class ModuleBase : ModuleBase<ICommandContext> { }
```
Comment (k-boyle) [inline src/Discord.Net.Commands/ModuleBase.cs]: Not a fan of this one.

File: src/Discord.Net.Commands/Attributes/RemarksAttribute.cs [hunk 3]
```diff
@@ -3,6 +3,9 @@
 namespace Discord.Commands
 {
     // Extension of the Cosmetic Summary, for Groups, Commands, and Parameters
+    /// <summary>
+    ///     An attribute that allows you to attach remarks to your command.
```
Comment (Still34) [inline src/Discord.Net.Commands/Attributes/RemarksAttribute.cs]: I would suggest rephrasing the summary for both the attributes. A summary should begin with a verb like all the other summaries in this lib.

References:
* https://docs.microsoft.com/en-us/dotnet/api/system.obsoleteattribute?view=netframework-4.7.2
* https://docs.microsoft.com/en-us/dotnet/api/system.flagsattribute?view=netframework-4.7.2

File: src/Discord.Net.Commands/ModuleBase.cs (hunk 2 above)
Comment (Still34) [reply src/Discord.Net.Commands/ModuleBase.cs]: Perhaps `Provides a base class for a command module to inherit from` or something along the lines?

**Review by Still34 — CHANGES_REQUESTED** [review]

**Review by Still34 — APPROVED** [review]

**Pull request merged**